x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/FollowerFailOverIT.java

```diff
@@ -164,6 +165,12 @@ public void testFollowIndexAndCloseNode() throws Exception {
         int maxOpsPerRead = followRequest.getParameters().getMaxReadRequestOperationCount();
         int maxNumDocsReplicated = Math.min(between(50, 500), between(maxOpsPerRead, maxOpsPerRead * 10));
         availableDocs.release(maxNumDocsReplicated / 2 + 1);
+        safeAwait(
```

Comment:
comment with context for the objective of this call, please.

Reply:
Which I think is that we want to wait for primary shards to be ready for indexing, to consistently accept the writes, before we then randomly stop a node and interrupt indexing, and potentially index shard recovery.